tools/depends/target/gnutls/Makefile

```diff
@@ -28,9 +28,9 @@ $(TARBALLS_LOCATION)/$(ARCHIVE):
 
 $(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
 	rm -rf $(PLATFORM)/*; mkdir -p $(PLATFORM)
-	cd $(PLATFORM); $(ARCHIVE_TOOL) $(ARCHIVE_TOOL_FLAGS) $(TARBALLS_LOCATION)/$(ARCHIVE)
 	cd $(PLATFORM); patch -p1 -i ../size-max.patch
 	cd $(PLATFORM); patch -p1 -i ../add-dl-as-private-lib.patch
+	cd $(PLATFORM); patch -p1 -i ../disable-hw-accel-aarch64.patch
```

Comment:
Why disable hardware acceleration for all aarch64 cpus?

Reply:
Good point